packages/apps/src/Menu/types.tsx

```diff
@@ -1,8 +1,8 @@
 // Copyright 2017-2020 @polkadot/apps authors & contributors
 // SPDX-License-Identifier: Apache-2.0
 
-import { IconName } from '@fortawesome/fontawesome-svg-core';
 import { Routes } from '@polkadot/apps-routing/types';
+import { IconName } from '@fortawesome/fontawesome-svg-core';
```

Comment:
This is also a `type { IconName }` import.

Reply:
I've applied separate rule for that import, now it works as it was supposed to